### PR TITLE
add tests for newlines in InlineText

### PR DESCRIPTION
**Pull request by srilq**
Closes: https://app.clubhouse.io/condenastinternational/story/7922

### CODE REVIEW

File: tests/InlineTextTest.php [hunk 1]
```diff
@@ -27,29 +24,53 @@ public static function expectedRenders()
...
+            ),
+            "expect empty string with subtype" => array(
+                new InlineText("", "~"),
+                ""
+            ),
+            "expect multiple lines of whitespace only to be preserved with spaces on the first line" => array(
+                new InlineText("  \n"),
+                "\n"
+            ),
+            "expect multiple lines of whitespace only to be preserved with spaces on the last line" => array(
```
Comment (graycodes) [inline tests/InlineTextTest.php]: I don't fully understand this test. Could it be named another way?

File: tests/InlineTextTest.php [hunk 2]
```diff
@@ -27,29 +24,53 @@ public static function expectedRenders()
                 new InlineText("  Hello world!  ", "¯\_(ツ)_/¯"),
                 "  ¯\_(ツ)_/¯Hello world!¯\_(ツ)_/¯  "
             ),
-            "expect only whitespace and subtype" => array(
+            "expect text with internal newlines" => array(
+                new InlineText("First\nSecond\n\n\nThird", ":"),
+                ":First:\n:Second:\n\n:Third:"
+            ),
```
Comment (graycodes) [inline tests/InlineTextTest.php]: I'm assuming we strip newlines >2 and this is expected.

Comment (srilq) [reply tests/InlineTextTest.php]: @graycodes yep
https://github.com/conde-nast-international/copilot-markdown-generator-php/blob/master/tests/TextTest.php#L34

File: tests/InlineTextTest.php (hunk 1 above)
Comment (srilq) [reply tests/InlineTextTest.php]: @graycodes Yeah, this is poorly named, taking a look now. (I named hundreds of tests in a short time period so I didn't really think them through!)

**Pull request merged**